api/benchmarks/tracer_bench.rb

```diff
@@ -31,13 +31,8 @@
     span.finish
   end
 
-  x.report 'start span with parent' do
-    span = tracer.start_span('test_span', with_parent: parent_span)
-    span.finish
-  end
-
   x.report 'start span with parent context' do
-    span = tracer.start_span('test_span', with_parent_context: parent_span.context)
+    span = tracer.start_span('test_span', with_parent: parent_span.context)
```

Comment:
This is not correct (it wasn’t correct before either). `with_parent:` wants a `Context`, but `parent_span.context` is a `SpanContext`.